ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTE5MzIwNjMyLjE3NDYzMTM0NzYifSwiYW5vbklkIjoiZTAwZDg4ZWUxZjA0MzAzMCJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zYWxvb24vcy1jbGFzcy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjAzMjAwMzYyMTExODQwMjYiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x557ff285f75a <unknown>
#1 0x557ff23124b0 <unknown>
#2 0x557ff22f8d86 <unknown>
#3 0x557ff22f769a <unknown>
#4 0x557ff22f7dff <unknown>
#5 0x557ff22f7d54 <unknown>
#6 0x557ff231669f <unknown>
#7 0x557ff23bb92a <unknown>
#8 0x557ff2389592 <unknown>
#9 0x557ff23af5e0 <unknown>
#10 0x557ff2389363 <unknown>
#11 0x557ff2355d63 <unknown>
#12 0x557ff23569c1 <unknown>
#13 0x557ff2824a6b <unknown>
#14 0x557ff2828951 <unknown>
#15 0x557ff280cb62 <unknown>
#16 0x557ff28294c4 <unknown>
#17 0x557ff27f113f <unknown>
#18 0x557ff284d6f8 <unknown>
#19 0x557ff284d8d6 <unknown>
#20 0x557ff285e5a6 <unknown>
#21 0x7f66eac9caa4 <unknown>
#22 0x7f66ead29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMjMwIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9JVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9zLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIn0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMTkzMjA2MzIuMTc0NjMxMzQ3NiJ9LCJhbm9uSWQiOiJlMDBkODhlZTFmMDQzMDMwIn0sInBlcmZvcm1hbmNlIjp7fSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6eyJQcm9kdWN0Ijp7Il9pZCI6IjIyMzAiLCJkaW1lbnNpb25zIjp7IkJhdW11c3RlclNob3J0IjpbIjIyMzAiXX19fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI2OTkyNzk3Mzc5OTg2NzkifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x557ff285f75a <unknown>
#1 0x557ff23124b0 <unknown>
#2 0x557ff22f8d86 <unknown>
#3 0x557ff22f769a <unknown>
#4 0x557ff22f7dff <unknown>
#5 0x557ff22f7d54 <unknown>
#6 0x557ff231669f <unknown>
#7 0x557ff23bb92a <unknown>
#8 0x557ff2389592 <unknown>
#9 0x557ff23af5e0 <unknown>
#10 0x557ff2389363 <unknown>
#11 0x557ff2355d63 <unknown>
#12 0x557ff23569c1 <unknown>
#13 0x557ff2824a6b <unknown>
#14 0x557ff2828951 <unknown>
#15 0x557ff280cb62 <unknown>
#16 0x557ff28294c4 <unknown>
#17 0x557ff27f113f <unknown>
#18 0x557ff284d6f8 <unknown>
#19 0x557ff284d8d6 <unknown>
#20 0x557ff285e5a6 <unknown>
#21 0x7f66eac9caa4 <unknown>
#22 0x7f66ead29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjExOTMyMDYzMi4xNzQ2MzEzNDc2In0sImFub25JZCI6ImUwMGQ4OGVlMWYwNDMwMzAifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc2Fsb29uL3MtY2xhc3Mvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI5MzU4NjA2MTE3NzIzMzA0IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x557ff285f75a <unknown>
#1 0x557ff23124b0 <unknown>
#2 0x557ff22f8d86 <unknown>
#3 0x557ff22f769a <unknown>
#4 0x557ff22f7dff <unknown>
#5 0x557ff22f7d54 <unknown>
#6 0x557ff231669f <unknown>
#7 0x557ff23bb92a <unknown>
#8 0x557ff2389592 <unknown>
#9 0x557ff23af5e0 <unknown>
#10 0x557ff2389363 <unknown>
#11 0x557ff2355d63 <unknown>
#12 0x557ff23569c1 <unknown>
#13 0x557ff2824a6b <unknown>
#14 0x557ff2828951 <unknown>
#15 0x557ff280cb62 <unknown>
#16 0x557ff28294c4 <unknown>
#17 0x557ff27f113f <unknown>
#18 0x557ff284d6f8 <unknown>
#19 0x557ff284d8d6 <unknown>
#20 0x557ff285e5a6 <unknown>
#21 0x7f66eac9caa4 <unknown>
#22 0x7f66ead29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9zLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ2ZWhpY2xlQ29kZSI6IncyMjMiLCJ2ZWhpY2xlQk0iOiIyMjMwIiwidmVoaWNsZVBJSGlzdG9yeSI6IjIyMzAiLCJ0aW1lc3RhbXAiOjE3NDYzMTM0ODE3MjAsImV4cGlyYXRpb24iOjEyMH19fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMTkzMjA2MzIuMTc0NjMxMzQ3NiJ9LCJhbm9uSWQiOiJlMDBkODhlZTFmMDQzMDMwIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9zLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIxNzU5MTYyNTcwMzI2MDk2IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x557ff285f75a <unknown>
#1 0x557ff23124b0 <unknown>
#2 0x557ff22f8d86 <unknown>
#3 0x557ff22f769a <unknown>
#4 0x557ff22f7dff <unknown>
#5 0x557ff22f7d54 <unknown>
#6 0x557ff231669f <unknown>
#7 0x557ff23bb92a <unknown>
#8 0x557ff2389592 <unknown>
#9 0x557ff23af5e0 <unknown>
#10 0x557ff2389363 <unknown>
#11 0x557ff2355d63 <unknown>
#12 0x557ff23569c1 <unknown>
#13 0x557ff2824a6b <unknown>
#14 0x557ff2828951 <unknown>
#15 0x557ff280cb62 <unknown>
#16 0x557ff28294c4 <unknown>
#17 0x557ff27f113f <unknown>
#18 0x557ff284d6f8 <unknown>
#19 0x557ff284d8d6 <unknown>
#20 0x557ff285e5a6 <unknown>
#21 0x7f66eac9caa4 <unknown>
#22 0x7f66ead29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'